packages/backend-plugin-api/src/services/definitions/PermissionsRegistryService.ts

```diff
@@ -59,7 +68,9 @@ export type PermissionsRegistryServiceAddResourceTypeOptions<
    * resolve conditional decisions except when requesting resources directly
    * from the plugin.
    */
-  getResources?(resourceRefs: string[]): Promise<Array<TResource | undefined>>;
+  getResources?(
+    resourceRefs: string[],
```

Comment:
This one is still meant to take just strings, not resource ref objects? Sorry maybe it's obvious but I was not sure about this part of the system

Reply:
Yep this is essentially a data loader. These refs aren't refs but something like entity string refs.

Reply:
oh that's not confusing at all :D